config/config.go

```diff
@@ -97,6 +97,9 @@ type Config struct {
 	Twilio struct {
 		Enable bool `public:"true" info:"Enables sending and processing of Voice and SMS messages through the Twilio notification provider."`
 
+		VoiceName     string `public:"true" info:"The Twilio voice to use for Text To Speech for phone calls."`
+		VoiceLanguage string `public:"true" info:"The Twilio voice language to use for Text To Speech for phone calls."`
```

Comment:
Are we using these in the UI outside of the admin config panel? If not, we can leave the `public` tag off. That is only intended to indicate fields that should be readable by non-admin users.

Reply:
not really.  removed the public tags.